conf/machine/imx6qdl-pico.conf

```diff
@@ -44,8 +45,13 @@ MACHINE_ESSENTIAL_EXTRA_RDEPENDS += " \
 
 MACHINE_EXTRA_RRECOMMENDS += " \
     bcm4339-nvram-config \
+    wpa-supplicant \
```

Comment:
This doesn't seem right as it is not required for the machine to work. Also, a device can connect to an open wifi without it. This should be added to the user image.